application/templates/base.html

```diff
@@ -2,44 +2,7 @@
 <html lang="en">
     <head>
         <title>My TODO list</title>
-            <style>
-                #tasks {
-                font-family: Arial, Helvetica, sans-serif;
-                border-collapse: collapse;
-                width: 100%;
-                }
-
-                #tasks td, #tasks th {
-                  border: 1px solid #ddd;
-                  padding: 8px;
-                }
-
-                #tasks tr:nth-child(even){background-color: #f2f2f2;}
-
-                #tasks tr:hover {background-color: #ddd;}
-
-                #tasks th {
-                  padding-top: 12px;
-                  padding-bottom: 12px;
-                  text-align: left;
-                  background-color: #04AA6D;
-                  color: white;
-
-                }
-                .button {
-                  background-color: #4CAF50; /* Green */
-                  border: none;
-                  color: white;
-                  padding: 15px 32px;
-                  text-align: center;
-                  text-decoration: none;
-                  display: inline-block;
-                  font-size: 16px;
-                }
-                .button1 {background-color: #f44336;} /* Red */
-                .button2 {background-color: #008CBA;} /* Blue */
-                .button0 {float: right}
-            </style>
+            <link rel='stylesheet' type='text/css' href='static/style.css'>
```

Comment:
is it proper way to connect static files to flask app?
check this:
[https://flask.palletsprojects.com/en/2.2.x/tutorial/static/](url)

Reply:
static files connection changed